packages/hs-react-ui/src/enums/colors.ts

```diff
@@ -9,6 +9,11 @@ enum colors {
 
   success = '#698F00',
   destructive = '#d00',
+
+  blueCharcoal25 = 'rgba(211, 214, 215, 0.25)',
+  blueCharcoal50 = 'rgba(211, 214, 215, 0.5)',
+  blueCharcoal75 = 'rgba(211, 214, 215, 0.75)',
+  blueCharcoal100 = 'rgba(211, 214, 215, 1)',
```

Comment:
these are grayDark with different opacities, not the headstorm colors that match these. We also want the users to be able to change these eventually and calling it "blue" and "charcoal" will be weird when they use a warm grey base.

I vote grayDark25, grayDark50, etc

Reply:
Sure - there's a comment in figma's styles above the color I used that said blueCharcoal, which is why I used that name here. Maybe figma should also be changed.

Reply:
Sounds good to me